### ISSUE TITLE
[cmc] Move docs from wiki to rst. (Trac #2390)

### ISSUE THREAD

**Issue by icecube-trac** [icecube-trac/tix2#2382]
The docs claim the "main documentation" is in the wiki.  We prefer docs that live next to (and evolve with) the code.

Migrated from https://code.icecube.wisc.edu/ticket/2390
```json
{
    "status": "closed", 
    "changetime": "2020-06-24T12:31:42", 
    "description": "The docs claim the \"main documentation\" is in the wiki.  We prefer docs that live next to (and evolve with) the code.", 
    "reporter": "olivas", 
    "cc": "", 
    "resolution": "fixed", 
    "_ts": "1593001902142004", 
    "component": "combo simulation", 
    "summary": "[cmc] Move docs from wiki to rst.", 
    "priority": "normal", 
    "keywords": "", 
    "time": "2019-12-18T23:16:24", 
    "milestone": "Autumnal Equinox 2020", 
    "owner": "brianclark", 
    "type": "cleanup"
}
```

**Comment by icecube-trac** [comment]
Trac update at 20200204T20:56:00: brianclark changed status from "new" to "accepted"

**Comment by icecube-trac** [comment]
Trac update at 20200204T21:07:04: 
* brianclark changed _comment0 from "fixed in commit 178616/IceCube" to "1580850535014995"
* brianclark changed status from "accepted" to "closed"
* brianclark changed resolution from "" to "fixed"
* brianclark commented:

> fixed in commit [178616/IceCube]

**Comment by icecube-trac** [comment]
Trac update at 20200624T12:31:42: 
* olivas changed milestone from "Vernal Equinox 2020" to "Autumnal Equinox 2020"
* olivas commented:

> Milestone Vernal Equinox 2020 deleted
